Add authentication_pool_size option to wazuh server api (#8287)
wazuh/wazuh-documentation: CHANGELOG.md

# [v4.11.2]

### Added

> - Added the ``authentication_pool_size`` option to customize the number of authentication processes in the Wazuh server API configuration. ([#8287](https://github.com/wazuh/wazuh-documentation/pull/8287)

Reviewer: ```suggestion
- Added the ``authentication_pool_size`` option to the Wazuh server API configuration section. ([#8287](https://github.com/wazuh/wazuh-documentation/pull/8287))
```
